services/openshift-service-impl/src/test/java/org/kontinuity/catapult/service/openshift/impl/OpenShiftServiceCdiIT.java

```diff
@@ -41,16 +42,18 @@ protected OpenShiftService getOpenShiftService() {
     public static WebArchive createDeployment() {
         // Import Maven runtime dependencies
         final File[] dependencies = Maven.resolver().loadPomFromFile("pom.xml")
-                .importRuntimeDependencies().resolve().withTransitivity().asFile();
+                .importRuntimeAndTestDependencies().resolve().withTransitivity().asFile();
         // Create deploy file    
         WebArchive war = ShrinkWrap.create(WebArchive.class)
                 .addPackage(Fabric8OpenShiftClientServiceImpl.class.getPackage())
                 .addPackage(OpenShiftServiceCdiIT.class.getPackage())
                 .addPackage(OpenShiftService.class.getPackage())
                 .addClass(OpenShiftServiceSpi.class)
+                .addAsResource("pipelinetemplate.json")
+                .addAsWebInfResource("META-INF/jboss-deployment-structure.xml", "jboss-deployment-structure.xml")
                 .addAsLibraries(dependencies);
         // Show the deployed structure
-        log.fine(war.toString(true)); 
+        System.err.println(war.toString(true)); 
```

Comment:
log.info if you just always wanna see it; the err stream doesn't get captured by external processes looking to stream from sysout.